components/meter/meter-mixin.js

```diff
@@ -5,6 +5,11 @@ export const MeterMixin = superclass => class extends LocalizeCoreElement(superc
 
 	static get properties() {
 		return {
+			/**
+			 * Hides the text visually
+			 * @type {boolean}
+			 */
+			labelHidden: { type: Boolean, attribute: 'label-hidden' },
```

Comment:
>  Should we actually be using `text-hidden` instead?

IMO, yes. Also you'll want to set this to `false` in the constructor.

Reply:
In regards to Danny's comment about "label" vs. "text" -- typically we've used "label" with form-like things and "text" everywhere else (e.g. `button-icon`). This feels more like "something else" than a form-like thing. Also if we were to change this to "label" there'd be a lot of places to update.